streams/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -167,7 +167,6 @@ public ByteBuffer subscriptionUserData(final Set<String> topics) {
             } else {
                 return new FutureSubscriptionInfo(
                     usedSubscriptionMetadataVersion,
-                    LATEST_SUPPORTED_VERSION + 1,
```

Comment:
This argument is never used, because the `latestSupportedVersion` of a `FutureSubscriptionInfo` is the current version. Alternatively, this argument may be used, but then line 352 (in the original version) should be changed to `buf.putInt(latestSupportedVersion);`.